sgrep/lib/generic_vs_generic.ml

```diff
@@ -1033,6 +1032,21 @@ and m_list__m_argument (xsa: A.argument list) (xsb: A.argument list) =
       (* can match more *)
       (m_list__m_argument ((A.Arg (A.Ellipsis i))::xsa) xsb)
 
+  (* dots: '...', can also match no argument *)
```

Comment:
maybe add a comment here saying interpolated strings are transformed in Call(Spedial(Concat, ...)
and we want patterns like f"...{$X}...", which are expanded to Call(Special(Concat, [L"..."; Id "$X"; L"..."])) to
match concrete code like f"foo{a}"

Reply:
Ok